src/examples/accordions/stepper/StepperDefault.tsx

```diff
@@ -0,0 +1,65 @@
+/**
+ * Copyright Zendesk, Inc.
+ *
+ * Use of this source code is governed under the Apache License, Version 2.0
+ * found at http://www.apache.org/licenses/LICENSE-2.0.
+ */
+
+import React, { useState } from 'react';
+import styled from 'styled-components';
+import { Stepper } from '@zendeskgarden/react-accordions';
+import { Button } from '@zendeskgarden/react-buttons';
+import { Row, Col } from '@zendeskgarden/react-grid';
+
+const StyledBtnRow = styled.div`
+  display: grid;
+  grid-auto-flow: column;
+  grid-gap: ${p => p.theme.space.sm};
+  justify-content: start;
+  margin: ${p => p.theme.space.sm} 0 0 0;
+`;
+
+const Example = () => {
+  const [step, setStep] = useState(0);
+
+  return (
+    <Row>
+      <Col>
+        <Stepper activeIndex={step}>
+          <Stepper.Step key={'step-1'}>
+            <Stepper.Label>Make good use of your location</Stepper.Label>
+            <Stepper.Content>
+              The success of your garden depends greatly on location.
+              <StyledBtnRow>
```

Comment:
What's the purpose of this wrapping element? It seems to be like it should be removed. It's currently a wrapping div that applies CSS grid, flex, and margin styling. Doesn't seem like a single button needs CSS grid or flex styling. 

Maybe there could be an inline style or a `StyledButton` to apply margin if necessary.

Reply:
Was thinking of having one styled wrapper for the buttons that could apply to each instead of inline styling throughout the example code or a few separate wrappers. If I inline style on the button with margin, [it doesn't break the line](https://share.getcloudapp.com/bLuRZEON) which looked weird to me.  Open to a different/preferred approach.

Reply:
Ohh I see, in this case, the floating text should be wrapped in a block element. It might look something like:

```jsx
<div>The success of your garden depends greatly on location.</div>
<Button>Next</Button>
```